cmake/inputs/patches/ep_gcssdk/build.patch

```diff
@@ -77,6 +100,8 @@ index 9cc916aed..a5d2680e8 100644
 +               -DBUILD_SAMPLES=${BUILD_SAMPLES}
 +               -DCMAKE_CXX_FLAGS=-fPIC
 +               -DCMAKE_C_FLAGS=-fPIC
++               -DOPENSSL_ROOT_DIR=${OPENSSL_ROOT_DIR}
++               -DGOOGLE_CLOUD_CPP_ENABLE_MACOS_OPENSSL_CHECK=OFF
```

Comment:
@ihnorton FYI I had to add this to get the CI to pass. The nested `google-cloud-cpp-common-project` has an identical env-var check to this project which was causing my earlier confusion.